data/json/items/gun/8x40mm.json

```diff
@@ -74,7 +74,11 @@
     "aim_speed": 4,
     "recoil": 100,
     "durability": 9,
-    "burst": 5,
+    "modes": [
+      [ "DEFAULT", "semi-auto", 1 ],
+      [ "BURST", "2 rd.", 2 ],
```

Comment:
Good stuff. I'd like to extend this to other appropriate items as it starts making burst fire much more useful